src/v3/schema/schema.test.ts

```diff
@@ -36,8 +36,26 @@ describe("schema/v3.0", () => {
         "https://www.w3.org/2018/credentials/v1 needs to be first in the list of contexts"
       );
     });
+    it("should be invalid when @context is a string", async () => {
+      // @context MUST be an ordered set in W3C VC data model, see https://www.w3.org/TR/vc-data-model/#contexts
+      const document = { ...cloneDeep(sampleDoc), "@context": "https://www.w3.org/2018/credentials/v1" };
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      // @ts-ignore
```

Comment:
@ts-expect-error

Reply:
thanks!!

Note to self: remember Laurent's wise words "one thing also with ts-ignore, while it's cool at the moment, it can hide future breaking stuff" 😃

Reply:
fixed in 41195c5